lib/dojos.js

```diff
@@ -962,14 +962,39 @@ exports.register = function (server, options, next) {
       }
     }
   }, {
-    method: 'POST',
-    path: options.basePath + '/dojos/accept-user-request',
-    handler: handlers.actHandlerNeedsUser('accept_user_request'),
+    method: 'PUT',
+    path: options.basePath + '/dojos/{dojoId}/request/{inviteToken}/user/{requestedByUser}',
+    handler: handlers.actHandlerNeedsUser('accept_user_request', ['dojoId', 'inviteToken', 'requestedByUser']),
     config: {
       auth: auth.apiUser,
-      description: 'accept request',
-      notes: 'accept request',
-      tags: ['api', 'dojos']
+      description: 'Accept request from an user to join a dojo',
+      notes: 'Accept request from an user to join a dojo',
+      tags: ['api', 'dojos'],
+      validate: {
+        params: {
+          dojoId: Joi.string(),
```

Comment:
@DanielBrierton just to explain why it's not required here, it's because the email verification still use a path w/o dojoId so that I can't provide the info to the API call.
It's not required anyway, but most likely kept for the form of the API call